ggshield/core/git_shell.py

```diff
@@ -202,6 +216,62 @@ def get_staged_filepaths(wd: Optional[str] = None) -> List[Path]:
     return [Path(path_str) for path_str in filepaths]
 
 
+def get_diff_files_status(
+    wd: Optional[str], ref: str, staged: bool = False, similarity: int = 100
```

Comment:
Since `wd` is optional, it should be after non-optional arguments and have a default value of `None`.